### ISSUE TITLE
Book edited in a chapter of an article written by two scholars

### ISSUE THREAD

**Issue by Doc73** [maieul/biblatex-bookinarticle#20]
Dear Maieul,
I am not sure of how to treat a particular case, where an ancient book is edited in a chapter of an article written by two authors. The only method I found is the ```collection```, but this method is not satisfactory, because it lacks the `journal` field.

Here is my bib sample:
```tex
@Bookinincollection{Collectio1891--1893,
  author     = {{Collectio~proverbiorum~cod.~Vat.~gr.~306}},
  pages      = {238--253},
  year       = {1891--1893},
  crossref   = {Cohn1891--1893},
  subtitle   = {cum recensione D2 fere concinens},
  titleaddon = {\bibstring{bypublisher} L. Cohn},
}

@Incollection{Cohn1891--1893,
  author    = {Cohn, Leolpold},
  title     = {Zur Ueberlieferung des alphabetischen Corpus},
  year      = {1891--1893},
  pages     = {224--267},
  crossref  = {Crusius1891--1893},
}

@Collection{Crusius1891--1893,
  author    = {Crusius, Otto and Cohn, Leolpold},
  title     = {Zur handschriftlichen Ueberlieferung, Kritik und Quellenkunde der Paroemiographen},
  year      = {1891--1893},
  number    = {Suppl. 6},
  pages     = {201--324},
  addendum  = {(= C.P.G. Suppl. Nr. IV)},
  journal   = {Philologus},
}

**Comment by maieul** [comment]
you mean that article have chapter ? strange.

in this case we need:
`inarticle` for the section of the article
`bookininarticle` for the book edited 

and again, normally you don't need the titleaddon = {\bibstring{bypublisher} L. Cohn} as you can use now the "editor" field

**Comment by Doc73** [comment]
Yes, the article has many chapters, each of them written by an author. I have often this sort of cases. :sob: 
Do you think to add the types `inarticle` and `bookininarticle`?

Don't consider my string ```titleaddon = {\bibstring{bypublisher} L. Cohn}```, that depends on my personal needs.

**Comment by maieul** [comment]
inarticle already exists in the package. I think I could find time to complet with bookininarticle in the two next weeks

**Comment by maieul** [comment]
but what in the case don't consider each individual chapter as an indivdual article ? It is what I do for example for "notes and remarks" in some journal

**Comment by Doc73** [comment]
Yes, I can consider your suggestion, but there are cases of chapter written by two author. Now I'm going to school, but in the afternoon I can reflect on your solution.

**Comment by maieul** [comment]
in this case, it is just an article with two authors�
Le 16 juin 2016 � 10:30, Domenico <notifications@github.com> a �crit :

> Yes, I can consider your suggestion, but there are cases of chapter written by two author. Now I'm going to school, but in the afternoon I can reflect on your solution.

**Comment by Doc73** [comment]
Dear Maieul, your idea to consider each individual chapter as an article is good, but please consider to add the new `bookininarticle`.
You can read an article of my type here: https://archive.org/details/bub_gb_jaINAAAAYAAJ (pag.  201)

**Comment by maieul** [comment]
I consider it� but I am very busy now

**Comment by maieul** [comment]
I start to manage this new entry. Please let me know if I have found the good text in the journal. https://github.com/maieul/biblatex-bookinarticle/blob/082813fbf7cf60d4b5364bea9f5c3b103f96f520/documentation/example-bookininarticle.bib

I am to tired/busy to try to understand german and greek.

**Comment by maieul** [comment]
Please also try the branch issue20.

Here joined the new handbook, where you could find specification for @bookininarticle and @bookininbook.

[biblatex-bookinother.pdf](https://github.com/maieul/biblatex-bookinarticle/files/459642/biblatex-bookinother.pdf)

**Comment by Doc73** [comment]
Dear Maieul, give me some days: I'm very very busy right now!
Only a question: I have to test issue20 and issue23 separately or they are connected each other?

**Comment by maieul** [comment]
for now, the branch "issue20" contain also the change of the branch "issue23".

**Comment by Doc73** [comment]
Ah, ok! I'll try issue20.
As for the articles, you could use my examples in my first post.

**Comment by maieul** [comment]
ok, 
https://github.com/maieul/biblatex-bookinarticle/blob/f714dc48175ed4457b7bbde7bce5cb58c211992f/documentation/example-bookininarticle.bib

should be ok and more relevant (ps: note the issue of the series field� I have discovered it not so many time ago)

I will publish next week, except if you find bug before ;-)

Le 7 sept. 2016 � 19:21, Domenico <notifications@github.com> a �crit :

**Comment by Doc73** [comment]
Ah, yes! I've already in my bibliographic db:
`@Article{Crusius1891--1893,
  author       = {Crusius, Otto and Cohn, Leolpold},
  journaltitle = {Philologus},
  pages        = {201--324},
  title        = {Zur handschriftlichen Ueberlieferung, Kritik und Quellenkunde der Paroemiographen},
  volume       = {6},
  year         = {1891--1893},
  addendum     = {(= C.P.G. Suppl. Nr. IV)},
  series       = {Suppl.},
}`

**Comment by Doc73** [comment]
Ok with branch #20. There is still an issue with #23

**Comment by maieul** [comment]
solved by 8751cc7
